specification/_types/aggregations/pipeline.ts

```diff
@@ -162,6 +162,7 @@ export class InferenceConfigContainer {
   regression?: RegressionInferenceOptions
   /** Classification configuration for inference. */
   classification?: ClassificationInferenceOptions
+  ner?: NerInferenceOptions
```

Comment:
The inclusion of this ner option here is a bit confusing to me, since IMO it implies NER is supported in the inference bucket aggregation, which AFAIK is not true per the docs https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-pipeline-inference-bucket-aggregation.html.

Reply:
I think these NLP properties *are* supported in [inference processors](https://www.elastic.co/guide/en/elasticsearch/reference/master/inference-processor.html) in the create pipeline APi and in the [create trained model API](https://www.elastic.co/guide/en/elasticsearch/reference/master/put-trained-models.html), so iMO it makes more sense to add these in https://github.com/elastic/elasticsearch-specification/blob/main/specification/ingest/_types/Processors.ts or https://github.com/elastic/elasticsearch-specification/blob/main/specification/ml/_types/TrainedModel.ts